### PR TITLE
Headline matches TOC now

### PR DESCRIPTION
**Pull request by goldenmeanie**
There was some inconsistency between the names of pages in the sidebar and the headline on that screen. There were also some odd singular/plural things. 

This has some overlap with https://github.com/department-of-veterans-affairs/vets-website/issues/3291

### CODE REVIEW

File: src/js/edu-benefits/utils/helpers.js [hunk 1]
```diff
@@ -3,7 +3,7 @@ import moment from 'moment';
 
 export const chapterNames = {
   veteranInformation: 'Veteran Information',
-  benefitsEligibility: 'Benefits Eligibility',
+  benefitsEligibility: 'Benefits',
```
Comment (akainic) [inline src/js/edu-benefits/utils/helpers.js]: Why are we removing `Eligibility` here? The url still has `benefits-eligibility`.

**Review by akainic — COMMENTED** [review]
LGTM minus one question!

File: src/js/edu-benefits/utils/helpers.js (hunk 1 above)
Comment (goldenmeanie) [inline src/js/edu-benefits/utils/helpers.js]: Has two sub-sections: Benefit Selection and Benefit Relinquishment. Benefits is the union.

**Comment by jkassemi** [comment]
Jenkins CI errors fixed in master, and should run properly after successful master merge.

**Comment by jcmeloni-usds** [comment]
@goldenmeanie   will you please clear up merge conflicts and get this in so we can clean up the PRs?  Thanks! cc @ayaleloehr

**Comment by goldenmeanie** [comment]
We're going to remove the TOC soon. Closing

**Pull request closed without merging**